SukiUI/Utilities/Effects/EffectBackgroundDraw.cs

```diff
@@ -33,10 +31,16 @@ protected override void EffectChanged(SukiEffect? oldValue, SukiEffect? newValue
             _transitionEndTime = TransitionSeconds + (float)Math.Max(0, TransitionTime);
```

Comment:
You can replace `TransitionSeconds` with `CompositionCustomVisualHandler.CompositionNow` timespan property, which is syncrhonised with render timer. Probably won't improve anything perf-wise though.